apps.js

```diff
@@ -0,0 +1,11 @@
+
+const newPic = document.createElement ('img') ///create new elm newPic 
+newPic.setAttribute('src,https://picsum.photos/id/1003/1181/1772');
```

Comment:
Don't forget to saprate the each paramater by comma like this
```suggestion
newPic.setAttribute('src','https://picsum.photos/id/1003/1181/1772');
```